src/jit/gentree.h

```diff
@@ -2671,7 +2671,7 @@ struct GenTreeCall final : public GenTree
 
 #ifdef FEATURE_READYTORUN_COMPILER
     // Call target lookup info for method call from a Ready To Run module
-    CORINFO_CONST_LOOKUP gtEntryPoint;
+    CORINFO_LOOKUP gtEntryPoint;
```

Comment:
I'm going to revert this change, and just pass whatever I need to the APIs separately